x-pack/legacy/plugins/siem/public/components/toasters/errors.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export class ToasterError extends Error {
+  public messages: string[];
+
+  constructor(messages: string[]) {
+    super(messages[0]);
```

Comment:
You can only surface the first error, but you can also do:

```ts
super(messages.join(', '));
```

And at least if someone prints out the error message or uses this incorrectly they would get all the errors and not the first one?

Reply:
This class already existed; I was trying not to alter its behavior so as not to introduce more bugs/diff.